src/LinuxTracing/ThreadStateManager.h

```diff
@@ -52,30 +52,35 @@ class ThreadStateManager {
   void OnNewTask(uint64_t timestamp_ns, pid_t tid, pid_t was_created_by_tid,
                  pid_t was_created_by_pid);
   [[nodiscard]] std::optional<orbit_grpc_protos::ThreadStateSlice> OnSchedWakeup(
-      uint64_t timestamp_ns, pid_t tid, pid_t was_unblocked_by_tid, pid_t was_unblocked_by_pid);
+      uint64_t timestamp_ns, pid_t tid, pid_t was_unblocked_by_tid, pid_t was_unblocked_by_pid,
+      bool wait_for_callstack = false);
```

Comment:
I think this should be called `has_wakeup_callstack` or `has_unblocking_callstack` (depending if we want to match the name of the other parameters or the names in the proto, although we should make the names of the other parameters and the names in the proto coincide).

Reply:
Done.